src/modules/commands/discord/urbandictionary.ts

```diff
@@ -0,0 +1,83 @@
+import { Command } from '@commands/CommandHandler'
+import got from 'got'
+import { isRecord } from '@app/validators/object'
+import camelcaseKeys from 'camelcase-keys'
+import discord from 'discord.js'
+
+type definition = {
```

Comment:
Types are conventionally written in PascalCase, i.e. please change to Definition.